Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -246,6 +248,27 @@ end MeasurableMul
 
 end Mul
 
+section Semigroup
+
+variable [Semigroup G] [MeasurableMul G]
+
+/-- The image of a left invariant measure under scalar multiplication is left invariant. -/
+@[to_additive]
```

Comment:
For each of these additive versions, can you write a docstring about "scalar addition"?